Dockerfile

```diff
@@ -1,44 +1,28 @@
 FROM gcr.io/google.com/cloudsdktool/cloud-sdk as base
+RUN apt-get update \
+    && apt-get --quiet install -y git bash curl jq zip \
+    && apt-get --quiet clean autoclean \
+    && apt-get --quiet autoremove --yes \
+    && rm -rf /var/lib/{apt,dpkg,cache,log}/
+    #&& apt-get --quiet install -y git bash bc curl jq python3 software-properties-common wget ca-certificates zip \
+ENV TERRAFORM_VERSION="1.4.6"
+ENV PATH="${PATH}:/usr/local/gcloud/google-cloud-sdk/bin"
+WORKDIR /root/aiinfra
+RUN curl -s "https://releases.hashicorp.com/terraform/index.json" \
```

Comment:
what is the purpose of this ?

Reply:
That gets the latest version of terraform! I forgot to delete the `ENV TF_VERSION` so that is probably pretty confusing. We can change this back if there's too much automation here.